pages/content/amp-dev/community/roadmap.html

```diff
@@ -2,86 +2,187 @@
 $title: Roadmap
 $order: 2
 $view: /views/custom.j2
-roadmap: !g.json /content/amp-dev/community/roadmap.json
+roadmap: !g.yaml /shared/data/roadmap.yaml
+
 ---
 {% do doc.styles.addCssFile('css/components/templates/roadmap.css') %}
-{% do doc.amp_dependencies.add('amp-bind', '0.1') %}
 
 <main class="ap--main ap-t-roadmap">
-  <div class="ap--container">
-    <div class="ap-o-roadmap">
-
-      <amp-state id="roadmapfilter">
-        <script type="application/json">
-          {
-            "activefilter": "all"
-          }
-        </script>
-      </amp-state>
-
-      {% do doc.styles.addCssFile('css/components/organisms/filter-bubbles-list.css') %}
-
-      <section class="ap--content">
-        <h1>{{ _('The AMP Project Roadmap') }}</h1>
-        <div class="ap-m-tip ap-m-tip-note">
-          <div class="ap-m-tip-content">
-            {% do doc.styles.addCssFile('css/components/molecules/tip.css') %}
-            <p>{{ _('This is a high level overview suitable for all audiences. For a more detailed developer view, <a href="https://github.com/ampproject/amphtml/projects/43">head to GitHub.</a>') }}</p>
-          </div>
+  <section class="ap--stage ap--container-fluid">
+    {% do doc.styles.addCssFile('css/components/organisms/stage.css') %}
+
+    <div class="ap-o-stage">
+      <div class="ap--container-fluid">
+        <div class="ap-o-stage-content">
+          <h1 class="ap-o-stage-content-headline">{{ _('The AMP Project Roadmap') }}</h1>
+        </div>
+        <div class="ap-o-stage-image">
+          {% do doc.icons.useIcon('icons/roadmap-header.svg') %}
+          <svg style="height: 80%; width: 100%;"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#roadmap-header"></use></svg>
         </div>
+      </div>
+    </div>
+  </section>
 
+  <section class="ap-t-roadmap-intro">
```

Comment:
Please replace the text in this section with the following for as long someone comes up with something better:

```html
The ongoing development of AMP is conducted in <a href="{{ g.doc('/content/amp-dev/community/roadmap.html', locale=doc.locale).url.path }}">working groups</a>, each with knowledge or interest in a specific area of AMP.

To make their work visible to the community all groups provide regular status updates with details of what they have accomplished and what's up next. 

Info: All of the updates listed here are actually issues on GitHub and we'd love you to join the conversation there!
```